test/sales_engine_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'minitest/pride'
+require 'minitest/autorun'
```

Comment:
@jbranham1 do you need require './test/test_helper' here instead?

Reply:
Fixed!